book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+
+If possible, consider co-developing plain language and Easy Read content alongside a community representative or person with lived experience. 
+If this is not possible for your event, ensure that core communications and event content are checked by using an external tool such as the [Hemingway App](https://hemingwayapp.com/). 
+You can also paste the URL of a website into a [Readability Checker](https://www.webfx.com/tools/read-able/) to check the content of your event's webpage. 
+There are more resources below that can help you prepare Plain Language content. 
+
+Aim to have all core event communications be understandable by a general audience, and written in Plain Language. This does not need to mean that your entire event is pitched to a general audience, however by making all general information understandable you will be more inclusive and more able to engage all potential attendees, from research staff, researchers, organizers, people with lived experience, and other professions associated with your event's content. 
+
+Using plain language for your communications can help all attendees understand and engage in your event. 
+
+Additionally, consider how you will be mindful of, and review, language choices to ensure inclusivity. 
+For example, avoid gender-specific terms and include language that recognizes diverse identities in your event information and content. 
+This can help acknowledge and respect the cultural differences of your attendees. 
+
+(cl-inclusive-events-comms-plain-language-resources)=
+#### Resources for Writing in Plain Language
+The Office for National Statistics in the UK has [published a short guide on writing with Plain Language](https://service-manual.ons.gov.uk/content/writing-for-users/plain-language), 
+and there are online tools such as the [Hemingway App](https://hemingwayapp.com/) which give you feedback on your writing to make it clearer. 
+If you have a Microsoft Word account, there is also a feature (and [instructions on how to use it](https://support.microsoft.com/en-us/office/get-your-document-s-readability-and-level-statistics-85b4969e-e80a-4777-8dd3-f7fc3c8b3fd2?ui=en-us&rs=en-us&ad=us#__toc342546555)) which provides the reading level of your document's content.
+
+(cl-inclusive-events-comms-plain-language-summaries)=
+#### Plain Language Summaries
+If your event has a large focus on research content, consider writing plain language summaries of core content so that people with lived experience 
+who do not have a background in research are able to engage in the sessions and understand the projects that are discussed. 
+
+Please refer to The Turing Way's chapter on {ref}`cm-lay-summaries` for more guidance.
+and you can view good examples of plain language summaries from the [AI for Multiple Long Term Conditions research projects](https://aim-rsf.github.io/Glossary-of-Terms/pages/aim-projects.html). 
+
+(cl-inclusive-events-comms-accessibility-guide)=
+## Content Accessibility Guidance 
```

Comment:
This section is also very detailed and I'm not sure if we should keep it at this level of detail here or link out to guidance instead? This could either be an external guide or we could consider making this its own page if it's an amalgamation of recommendations?

Reply:
I've had a think about some of these pieces and I think it would make sense to expand in some places, and give a dedicated subsection to others. I'll capture my thoughts in more detail in the core PR #3307. 

For now, I will go through this Communications subsection, action some of your suggestions, and see what pieces should shift into their own new subsection.

Reply:
okay!